pkg/api/util/db.go

```diff
@@ -292,6 +292,17 @@ func CreateFileShareSnapshotDBEntry(ctx *c.Context, in *model.FileShareSnapshotS
 		log.Error(errMsg)
 		return nil, errors.New(errMsg)
 	}
+	reg, err := regexp.Compile("^[a-zA-Z0-9 _-]+$")
+	if err != nil {
+		errMsg := fmt.Sprintf("regex compilation for file share snapshot validation failed")
+		log.Error(errMsg)
+		return nil, errors.New(errMsg)
+	}
+	if reg.MatchString(in.Description) == false {
+		errMsg := fmt.Sprintf("invalid file share snapshot description it only contain english char and number  : %v", in.Name)
```

Comment:
message is slightly confusing because we are validating the description but displaying in.Name at end.
May be we can reframe the message like, (this is just example):
fmt.Sprintf("fileshare snapshot creation failed, because description has some special chars: %v. Description only support english char and number", in.Description)

Remaining code looks good.

Reply:
Thanks @PravinRanjan10 i resolved the issue